experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/unboxed.spec.ts

```diff
@@ -211,6 +279,254 @@ describe("unboxed unit tests", () => {
 				assert.equal(unboxed.value, true);
 			});
 		});
+
+		describe("Value field", () => {
+			it("Boolean", () => {
+				const { fieldSchema, context, cursor } = createValueLeafTree(
+					ValueSchema.Boolean,
+					true,
+				);
+				assert.equal(unboxedField(context, fieldSchema, cursor), true);
+			});
+
+			it("Number", () => {
+				const { fieldSchema, context, cursor } = createValueLeafTree(
+					ValueSchema.Number,
+					42,
+				);
+				assert.equal(unboxedField(context, fieldSchema, cursor), 42);
+			});
+
+			it("String", () => {
+				const { fieldSchema, context, cursor } = createValueLeafTree(
+					ValueSchema.String,
+					"Hello world",
+				);
+				assert.equal(unboxedField(context, fieldSchema, cursor), "Hello world");
+			});
+
+			// TODO: Fluid Handle
+
+			it("Struct", () => {
+				const builder = new SchemaBuilder("test");
+				const stringLeafSchema = builder.leaf("string", ValueSchema.String);
+				const booleanLeafSchema = builder.leaf("boolean", ValueSchema.Boolean);
+				const structSchema = builder.struct("struct", {
+					foo: SchemaBuilder.fieldValue(stringLeafSchema),
+					bar: SchemaBuilder.fieldSequence(booleanLeafSchema),
+				});
+				const rootSchema = SchemaBuilder.field(FieldKinds.value, structSchema);
+				const schema = builder.intoDocumentSchema(rootSchema);
+
+				const initialTree = {
+					foo: "Hello world",
+					bar: [true, false, true],
+				};
+
+				const { context, cursor } = initializeTreeWithContent(schema, initialTree);
+
+				const unboxed = unboxedField(context, rootSchema, cursor);
+
+				assert(unboxed !== undefined);
+				assert.equal(unboxed.foo, "Hello world");
+				assert.equal(unboxed.bar.length, 3);
+				assert.equal(unboxed.bar.at(0), true);
+				assert.equal(unboxed.bar.at(1), false);
+				assert.equal(unboxed.bar.at(2), true);
+			});
+
+			it("Recursive struct", () => {
```

Comment:
Recursive types tend to end up special at the compile time typing layer, but I don't think they should have any special unwrapping unless you look at a type that actually unwrapps. A recursive field node for example might be slightly interesting here, though thats more of an integration test (and a field node test) than unboxedField test.